### DISCUSSION TITLE
1000 error(s) occurred while feeding the table TABLENAME

### DISCUSSION THREAD

**Discussion by yerol89** [atoti/atoti#620] [Q&A] [answered]
Hello Everyone,

When I try to read csv using the following code snippet, I get the following warning.

`session_name = session.read_csv('data/path/path/**.csv',table_name='tablename', keys=["A", "B"])`

WARNING: 
**1000 error(s) occurred while feeding the table table_name. Check the table's loading_report for more details.**

Anyone faced with this issue before?
Thanks.

**Comment by tibdex** [comment, marked as answer]
If you do `session.tables["tablename"].loading_report` you should get some information about what is wrong with the data loading. A more generic way to investigate errors (i.e. not limited to data loading) is to look at the logs that are available at `session.logs_path`.

By the way, `Session.read_csv()` returns a `Table`, not a `session_name`.

**Comment by HuifangYeo** [comment]
Hi @tibdex,

in v0.7.0, `loading_report` has been made private. What would be the alternative means for users to check their loading error?

**Reply by Skyscimitar** [reply]
For now you can continue using the private `loading_report`, we are working on a better solution for accessing loading information for tables which will be introduced in an upcoming release.
